src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -166,70 +168,82 @@ class TransferBalancePage extends React.Component {
         const isButtonDisabled = !isTransferable || !selectedAccount;
         const errorMessage = !_.isEmpty(this.props.walletTransfer.errors) ? _.chain(this.props.walletTransfer.errors).values().first().value() : '';
 
+        const shouldShowTransferView =
+            PaymentUtils.hasExpensifyPaymentMethod(this.props.cardList, this.props.bankAccountList) && lodashGet(this.props.userWallet, 'tierName', '') === CONST.WALLET.TIER_NAME.GOLD;
```

Comment:
lodashGet for the tierName is not necessary, because the wallet defaults to an empty object, as I said before.
![image](https://github.com/Expensify/App/assets/26260477/51740664-9b33-4e9b-8ace-dd9c48c9a425)

Reply:
Got it, done @neil-marcellini

Reply:
I thought you bumped into an `undefined` error, so I suggested using `lodashGet` 😅